src/logion/controllers/adapters/verifiedthirdpartyadapter.ts

```diff
@@ -24,7 +24,7 @@ export class VerifiedThirdPartyAdapter {
     ) {}
 
     async nominatedParties(locRequestId: string): Promise<VerifiedThirdPartyView[]> {
```

Comment:
* This method, the attribute `nominatedParties` in view `LocRequestView` (`schemas.json`) and should also be renamed to `selectedParties`.
* And similar change is required in `logion-api`.

Reply:
Done. I still have to handle `logion-api`.